ui/apps/platform/src/ConsolePlugin/UserPermissionProvider.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { UserPermissionContext } from 'hooks/usePermissions';
+
+import { fetchUserRolePermissions } from 'services/RolesService';
+import useRestQuery from 'hooks/useRestQuery';
+
+export default function UserPermissionProvider({ children }: { children: React.ReactNode }) {
```

Comment:
Off the subject, what are pro and con of `React.ReactNode` versus `import type { ReactNode }` looking ahead to possible future when `import React` becomes unnecessary.